pkg/cvo/updatepayload.go

```diff
@@ -112,7 +122,7 @@ func (r *payloadRetriever) RetrievePayload(ctx context.Context, update configv1.
 
 	// download the payload to the directory
 	var err error
-	info.Directory, err = r.targetUpdatePayloadDir(retrieveCtx, update)
```

Comment:
This is bugged.  It couples the lookup timeout to the payload retrieval timeout, causing guaranteed failures when verifies timeout. This prevents force from working without signatures.  Before reintroducing, you should write a test.